libs/ng-morph/ng/component/add-provider-to-component.ts

```diff
@@ -0,0 +1,13 @@
+import { ClassDeclaration } from 'ts-morph';
+import { pushToArrayProperty } from '../helpers/push-to-array-property';
+
+export function addProviderToComponent(
+  classDeclaration: ClassDeclaration,
+  provider: string,
+  { unique }: { unique: boolean } = { unique: false }
```

Comment:
Мне кажется так будет лучше

```suggestion
  { unique = false }: { unique?: boolean } = { }
```